stl/inc/ranges

```diff
@@ -3409,6 +3451,9 @@ namespace ranges {
     } // namespace views
 
     // clang-format off
+    template <auto> // _Require_constant<E> is a valid template-id iff E is a constant expression of structural type
+    struct _Require_constant;

```

Comment:
Nitpick: This is the right location as it's needed for `_Tiny_range` immediately below, but does this need to be within the `// clang-format off`? I believe it could be moved just outside.